src/web3-utils.js

```diff
@@ -118,6 +119,9 @@ export function getUnknownBalance() {
 
 // Returns an identifier for the provider, if it can be detected
 export function identifyProvider(provider) {
+  if (provider && isElectron()) {
```

Comment:
Note that there's not a really good way to check for if we're connected to frame yet (_could_ try checking against `localhost:1234`).